cmd/rad/cmd/root.go

```diff
@@ -182,6 +183,23 @@ func Execute() error {
 	return nil
 }
 
+func initTracer() (func(context.Context) error, error) {
+	// Intialize the tracer provider
+	tp := sdktrace.NewTracerProvider(
+		sdktrace.WithSampler(sdktrace.AlwaysSample()),
+		sdktrace.WithResource(resource.NewWithAttributes(
+			semconv.SchemaURL,
+			semconv.ServiceNameKey.String(serviceName),
+		)),
+	)
+
+	// Set the tracer provider as "global" for the CLI process.
+	otel.SetTracerProvider(tp)
+	otel.SetTextMapPropagator(propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}))
+
+	return tp.Shutdown, nil
```

Comment:
I decoupled this from the server-side code in `pkg/components/trace`. We should avoid referencing server-side code in the CLI.